manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.2.0 using InputObject 🤖 $debug=QUSU.CRLF..7-2-7
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: OpenShot.OpenShot
+PackageVersion: 3.0.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- .osp
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86.exe
+  InstallerSha256: ED9FEFD35989B512E962DB7207187B3AE4E94CB2DCD4C233443C03BA180C86B6
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86_64.exe
+  InstallerSha256: 39B5AC64311FE5DE96CDF1C6660BD6A35EF1B1213EE48047E48932CABA2E0E91
```

Comment:
```suggestion
```

Reply:
Removing the x86 installer, as it appears to require x64 architecture, and having two x64 entries is causing the manifest to fail validation.